### ISSUE TITLE
Investigate noderify for faster cli loading times

### ISSUE THREAD

**Issue by scottcorgan** [cmd-js/cmd#3]
Noderify, creates one bundle file, avoiding the delay when loading npm modules synchronously. Not sure how much it will actually help, but I know that as cli apps get bigger, the execution gets slower.

https://github.com/dominictarr/noderify

**Comment by christiansmith** [comment]
That looks promising.

**Comment by scottcorgan** [comment]
@christiansmith have you noticed any lag in executing commands in your cli?

**Comment by christiansmith** [comment]
Yes, but it's all network latency so far. I've been concerned that we'll end up with something bloated as we continue building out.
